dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/DeadZone.mo

```diff
@@ -0,0 +1,93 @@
+within Dynawo.NonElectrical.Blocks.NonLinear;
+
+/*
+* Copyright (c) 2024, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite
+* of simulation tools for power systems.
+*/
+
+block DeadZone "Dead zone, bypassed if the limits are equal"
+  extends Modelica.Blocks.Interfaces.SISO;
+
+  parameter Real uMax(start = 1) "Upper limit of dead zone";
+  parameter Real uMin = -uMax "Lower limit of dead zone";
+
+equation
+  assert(uMax >= uMin, "DeadZone: limits must be consistent. However, uMax (=" + String(uMax) + ") < uMin (=" + String(uMin) + ")");
+
+  if uMax == uMin then
```

Comment:
To be tested with this instead
```suggestion
  if uMax == 0 and uMin == 0 then
```

Reply:
It works but it slows the simulation because of mode changes, for one example, 36 s instead of 27 s.

Reply:
For 100s simulation?

Reply:
Yes.